src/common/driverless_msgs/package.xml

```diff
@@ -7,9 +7,6 @@
     <maintainer email="team@qutmotorsport.com">QUT Motorsport</maintainer>
     <license>MIT</license>
 
-    <depend>std_msgs</depend>

```

Comment:
we should move towards having the actual set of requirements for each package in here and in conda requirements